.github/scripts/get_workflow_type.py

```diff
@@ -0,0 +1,98 @@
+import json
+from argparse import ArgumentParser
+from typing import Any
+
+from github import Auth, Github
+from github.Issue import Issue
+
+
+WORKFLOW_TYPE_LABEL = "label"
```

Comment:
Is it possible to add comments here what different workflow TYPES mean and how are used in context of the ARC ?
WORKFLOW_TYPE_RG - Use only arc runners ? What does RG stands for ?
WORKFLOW_TYPE_BOTH - Use ARC runners and old runners 
WORKFLOW_TYPE_LABEL - Use Old runners

Reply:
Thanks for pointing out the confusion. @DanilBaibak, how about we change the names to be:
RG -> ARC
LABEL -> CLASSIC

Reply:
We have two types of workflow:
* one uses [label](https://github.com/pytorch/pytorch/blob/main/.github/workflows/_linux-build-label.yml#L77) for runs-on;
* another uses [groups](https://github.com/pytorch/pytorch/blob/main/.github/workflows/_linux-build-rg.yml#L78) for runs-on;

So, "rg" refers to [runner groups](https://github.com/organizations/pytorch/settings/actions/runner-groups/51?page=1).

Reply:
@ZainRizvi, we already use "rg" and "label" for our workflows. Let me add comments and keep the naming the same. Anyway, we will only use this for the migration period and then leave only linux-build and linux-test.

Reply:
@atalman, please scroll a bit lower. I [have already added comments](https://github.com/pytorch/pytorch/pull/125680/files#diff-526206000667d38040f79b3c29a1f0e9246dee1bd14bf9fe3cb9b28bed8ae5fcR59-R77) about how each workflow type will impact users.

Reply:
@DanilBaibak yes I saw this. I think we can remove all comments lower. If we can make workflow TYPES more clear and with some comments where each type used.